image-types/rhel8/amazon-ec2.md

```diff
@@ -0,0 +1,48 @@
+# Amazon EC2 Image
+
+Owner: *None*
+
+This image is meant to be used in [Amazon Elastic Compute Cloud (EC2)][ec2], a
+popular cloud computing platform. It has to follow Amazon’s
+[guidelines][guidelines] and [requirements][requirements] for shared AMIs.
+
+## File Format
+
+EC2 uses Amazon Machine Images (AMIs) internally, which can only be created
+inside EC2. An image in a standard format (ova, vmdk, vhd/x, or raw) must be
+imported from S3 storage. *osbuild-composer* generates this image type in the
+VHDX format, because it is fairly modern, widely used, and uses less space than
+RAW images.
+
+The default image size is 6 GB.
+
+## Architectures
+
+Limited to `x86_64` and `aarch64`, because those are available on EC2.
```

Comment:
bike-shedding, but not only are they available, but they are _the only ones_ available

Reply:
You are right! All these beautiful reviews will get dismissed when I push again, so I won't do that and fix it in the next time I touch this file.